CMakeLists.txt

```diff
@@ -112,6 +112,10 @@ IF (HAVE_STRUCT_STAT_NSEC OR WIN32)
 	OPTION( USE_NSEC		"Care about sub-second file mtimes and ctimes"	ON  )
 ENDIF()
 
+IF (NOT USE_OPENSSL)
+	OPTION( USE_MBEDTLS     "Link with and use mbedTLS library"             ON )
```

Comment:
we should default this to off, otherwise on darwin it'll try to use both the apple crypto and mbedtls if found